blocks/v2-social-block/v2-social-block.js

```diff
@@ -25,19 +32,35 @@ const TEMPLATE_LINK_CONFIGS = [
   ['fb', 'https://www.facebook.com/sharer/sharer.php?u='],
 ];
 
-const buildTemplateBlock = (links) => {
-  const fragment = docRange.createContextualFragment(`
-    <ul class='${CLASSES.list}'>
-      ${links.map((link) => `
-        <li class='${CLASSES.listItem}'>
-          <a href='${link[1]}${window.location.href}' target='_blank'>
-            <span class='icon icon-${link[0]}'></span>
-          </a>
-        </li>
-        `).join('')}
+const buildTemplateBlock = (links, hasAttribution) => {
+  const attributionDate = getMetadata('attribution-date');
+  const attributionAuthor = getMetadata('attribution-author');
+
+  const attributionContent = hasAttribution ? `
+    <div class="${CLASSES.attributionWrapper}">
+      <p class="${CLASSES.attributionDate}">${attributionDate || ''}</p>
```

Comment:
Should the paragraph exist if attributionDate is not defined?
Same for the attributionAuthor